docs/repo-docs/reference/boundaries.mdx

```diff
@@ -0,0 +1,28 @@
+---
+title: boundaries
+description: API reference for the `turbo boundaries` command
+---
+
+import { ExperimentalBadge } from '#/components/experimental-badge';
+import { Callout } from '#/components/callout';
+
+<ExperimentalBadge>Experimental</ExperimentalBadge>
+
+Boundaries ensure that caching and other Turborepo features work correctly by checking for package manager Workspace violations.
```

Comment:
maybe we should just say "Turborepo works correctly" because if you break boundaries a lot of stuff won't work